config/locales/idv/fr.yml

```
@@ -162,10 +162,10 @@ fr:
       sessions:
         no_pii: SITE DE TEST - N’utilisez pas de véritables données personnelles (il
           s’agit d’une démonstration seulement) - SITE DE TEST
-        read_more_encrypt: En savoir plus sur la façon dont login.gov protège vos
+        read_more_encrypt: En savoir plus sur la façon dont Login.gov protège vos
           informations personnelles
-        review_message: Lorsque vous entrez à nouveau votre mot de passe, login.gov
-          crypte vos données pour vous assurer que personne ne peut y accéder.
+        review_message: Lorsque vous entrez à nouveau votre mot de passe, Login.gov
+        crypte vos données pour vous assurer que personne ne peut y accéder.
```

I think this line needs to be indented, YAML parse error from CirclecI:

```
rake aborted!
I18n::InvalidLocaleData: can not load translations from /home/circleci/identity-idp/config/locales/idv/fr.yml: #<Psych::SyntaxError: (<unknown>): could not find expected ':' while scanning a simple key at line 168 column 9>
/home/circleci/identity-idp/vendor/bundle/ruby/2.7.0/gems/i18n-1.8.10/lib/i18n/backend/base.rb:245:in `rescue in load_yml'
```

```suggestion
          crypte vos données pour vous assurer que personne ne peut y accéder.
```